frontend/vue/components/CodeExercise/ExerciseActionsBar.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="exercise-actions-bar">
+    <button
+      v-if="runEnabled && !isRunning"
+      class="exercise-actions-bar__button exercise-actions-bar__button_run"
+      @click="run()"
+    >
+      Run
+    </button>
+    <button
+      v-if="gradeEnabled && !isRunning"
+      class="exercise-actions-bar__button"
+      @click="grade()"
+    >
+      Grade
+    </button>
+    <div v-if="isRunning" class="exercise-actions-bar__running-indicator">
+      <bx-loading class="exercise-actions-bar__running-indicator__icon" assistive-text="Running" type="small" />
+      <span class="exercise-actions-bar__running-indicator__label"> Running </span>
```

Comment:
Similar comment regarding translations, something like:

```
{{ $translate('Running') }}
```

then make sure the strings are defined in `strings.yaml` file